brand/admin.py

```diff
@@ -349,6 +352,19 @@ def num_linked(self, obj):
 
     inlines = [CommentaryInline, BrandFeaturesInline, DatasourceInline]
 
+    def save_model(self, request, obj, form, change):
+        """
+        This function is to create object of commentary model when default values are
+        provided by user in admin portal and save the data in respective database.
+        """
+        super().save_model(request, obj, form, change)
+        try:
+            obj.commentary
+        except ObjectDoesNotExist as e:
+            commentary_obj = Commentary.objects.create(brand_id=obj.id)
+            obj.commentary = commentary_obj
+            obj.save()
```

Comment:
Curious about why you need to add this.  Was the form not saving commentary properly?

Reply:
Yes, commentary object wouldn't get created when user enters all default values. So I added this logic.